src/app/shared/components/template/components/text-bubble/text-bubble.component.scss

```diff
@@ -119,3 +133,21 @@ $imageSize: 64px;
     max(var(--b), 100% - var(--p) - var(--h) * tan(var(--a) / 2)) 0
     max(var(--b), var(--p) - var(--h) * tan(var(--a) / 2)) / 0 0 var(--h) 0;
 }
+
+// For the speaker name
+.speaker-name {
+  position: absolute;
+  bottom: -55px;
+  left: 78px;
+  color: var(--ion-color-gray-700);
+}
+.container[data-position="right"] {
+  .speaker-name {
+    text-align: right;
+    right: 80px;
```

Comment:
nit(non-blocking): Just to quickly check, will this behave as expected when using a `right-to-left` language?  Not sure in this case whether the text or avatar would be on the left/right (might be good to double-check)